app/atoms/qr-scanner.ts

```diff
@@ -161,6 +161,25 @@ export const removeScannedItemsByAssetIdAtom = atom(
   }
 );
 
+// Remove kits based on kit id
+export const removeScannedItemsByKitIdAtom = atom(
```

Comment:
This is fine, i guess, but we dont do this anywhere else, in any other drawer so it creates a consistency. If we add this we should implement it everywhere.